compute/apiv1/smoke_test.go

```diff
@@ -0,0 +1,290 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// To run these tests, set GCLOUD_TESTS_GOLANG_PROJECT_ID env var to your GCP projectID
+
+package compute
+
+import (
+	"cloud.google.com/go/internal/testutil"
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: Let's avoid pulling in a new dependency. In general we follow [Useful Test Failures](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures).